sdcm/nemesis.py

```diff
@@ -2223,6 +2243,16 @@ def disrupt_delete_by_rows_range(self):
 
         self.delete_range_in_few_partitions(ks_cf, partitions_for_exclude)
 
+    def disrupt_delete_with_timestamp(self):
```

Comment:
I don't think this case is so unique to deserve a special nemesis of its own.
I commented that on the task as well.
We already have other "delete nemesis", you can just add the timestamp there as a random choice.

Reply:
Done, using "delete_half_partition" as base.